### DISCUSSION TITLE
Original Roller Coaster Tycoon Transparent Water Needed

### DISCUSSION THREAD

**Discussion by mapplayer** [OpenRCT2/OpenRCT2#20542] [Feature requests]
I Need The Original Transparent Water Of The Original RCT into OPEN RCT2?

Just Like You Guys Have The Semi-Transparent And Opaque Water.

There Need To Be 3 Types Of Water Into OPENRCT2 

1.Semi-Transparent Water (You Already Have)

2.Opaque Water (You Already Added it )

3.Transparent Water (Not Added Yet)

 

Bellow Is An Example Of How It Looks!

I Have Many Beautiful Parks I Can Post Into The Website, Please Help Me Out With Including This Option Into The Game!

Since the View Option Only Allows For 2 Types Of Water I Can Recommend You Put An Water Tab Into The Options Menu And Have The Ability To change it Into One Of The 3 Types Of Water.

Please Add The Original Transparent Water Option.

The Difference is The The Original Transparent Water Of The Original Roller Coaster Tycoon Is More Transparent And More Clear Than The Glittering Semi Transparent Water Of OPENRCT2 Just like Grass Is Different From Sand The Transparent Water Is A Feature That Is Missing From OPENRCT2 and Therefore those graphics Should Be Added And Integrated Into The New Game, Without Of Course Disrupting Or Omitting The Semi-Transparent Or Opaque Water Of OPENRCT2.

**Comment by ZehMatt** [comment]
You haven't provided the example.

**Reply by spacek531** [reply]
Transparent water I believe is the same as semi-transparent water with the animated pixels made transparent, but I could be wrong.

**Reply by ZehMatt** [reply]
So this is what it looks like with the Underwater View enabled 
![image](https://github.com/OpenRCT2/OpenRCT2/assets/5415177/90b78673-5d9d-498d-bf03-d32d99a85c51)

**Comment by TheClaudeQC** [comment]
You mean that you see correctly what's going on underneat whiteout the blue being in the way, right?
